### PR TITLE
add sentry to ide extension

### PR DESCRIPTION
**Pull request by felixhaeberle**
this PR aims to integrate sentry with the ide extension to have better error observability.

### CODE REVIEW

File: inlang/source-code/ide-extension/src/services/error-monitoring/implementation.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import * as Sentry from "@sentry/node"
+import { version } from "../../../package.json"
+import { isProduction } from "./env.js"
+
+export function initErrorMonitoring() {
+	Sentry.init({
+		dsn: "https://paste-sentry-url.here",
```
Comment (felixhaeberle) [inline inlang/source-code/ide-extension/src/services/error-monitoring/implementation.ts:7]: replace sentry url here after creating the project in sentry
cc @samuelstroschein

File: inlang/source-code/ide-extension/src/services/error-monitoring/implementation.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { isProduction } from "./env.js"
 
 export function initErrorMonitoring() {
 	Sentry.init({
-		dsn: "https://paste-sentry-url.here",
+		dsn: "https://ba058608bbf997cfe6c5f8d093d31d5b@us.sentry.io/4506700407177216",
```
Comment (samuelstroschein) [inline inlang/source-code/ide-extension/src/services/error-monitoring/implementation.ts:7]: @felixhaeberle is this deactiavated for dev mode? you don't want to spam sentry. also this should have been an env variable

Comment (felixhaeberle) [reply inlang/source-code/ide-extension/src/services/error-monitoring/implementation.ts:7]: i copied the setup you did for the CLI, so I thought it was on purpose

Comment (samuelstroschein) [reply inlang/source-code/ide-extension/src/services/error-monitoring/implementation.ts:7]: oh haha lol

**Pull request merged**